Manually curated lists of software, publications and other resources related to the analysis of HDX-MS data.

## Open software for analysis of the HDX-MS data

Open, free and maintained tools for analysis of HDX-MS data:

1. DECA: [publication](https://doi.org/10.1074/mcp.TIR119.001731); [repository](https://github.com/komiveslab/DECA).
1. deMix: [publication](https://doi.org/10.1038/s41598-019-39512-8); access on request.
1. Deuteros 2.0: [publication](https://doi.org/10.1093/bioinformatics/btaa677); [repository](https://github.com/andymlau/Deuteros_2.0). Version 1.0: [publication](https://doi.org/10.1093/bioinformatics/btz022); [repository](https://github.com/andymlau/Deuteros).
1. ExMS2: [publication](https://doi.org/10.1021/acs.analchem.9b01682); MATLAB package.
 EXMS: [publication](https://doi.org/10.1007/s13361-011-0236-3); MATLAB package.
1. HaDeX: [publication](https://doi.org/10.1093/bioinformatics/btaa587); [repository](https://github.com/hadexversum/HaDeX), [web server](https://hadex.mslab-ibb.pl/).
1. HDXanalyzer [publication](https://doi.org/10.1186/1471-2105-12-S1-S43); RPY package.
1. HDX-Viewer: [publication](https://doi.org/10.1093/bioinformatics/btz550); [web-server](https://masstools.ipbs.fr/hdx-viewer); [repository](https://github.com/david-bouyssie/hdx-viewer).
1. Hexicon 2: [publication](https://dx.doi.org/10.1007%2Fs13361-014-0850-y); access on request.
1. HD-eXplosion: [publication](https://doi.org/10.1093/bioinformatics/btaa892); [web-server](http://hd-explosion.utdallas.edu/); [repository](https://github.com/HD-Explosion).
1. HX-Express v2: [publication](https://dx.doi.org/10.1007%2Fs13361-013-0727-5); [availability](https://www.hxms.com/HXExpress/).
1. MEMHDX: [publication](https://doi.org/10.1093/bioinformatics/btw420); [web server](http://memhdx.c3bi.pasteur.fr/).
1. QUDeX-MS [publication](https://doi.org/10.1186/s12859-014-0403-1); [stand-alone tool](https://sourceforge.net/projects/qudex-ms/).

## Commercial software for analysis of the HDX-MS data

Maintained commercial tools for analysis of HDX-MS data:

## Deprecated tools for analysis of the HDX-MS data

Non-maintained (no updates for at least five years) tools for analysis of HDX-MS data:

## Repositories

Scripts and data for HDX-MS data analysis:

1. HDX Simulations:
 [Kan ZY](https://github.com/kanzy/HX-MS-Simulations), [Robert Bradshaw](https://github.com/rtb1c13/calc_hdx).
1. HDXrate: [repository](https://github.com/Jhsmit/HDXrate).
1. H/D exchange rate excel spreadsheets: [Englander group website](http://hx2.med.upenn.edu/download.html).
 
## High-resolution HDX-MS

Publications and software for HR HDX-MS analysis:

1. Babic et al., 2019: [publication](https://doi.org/10.1002/rcm.8460). Follow-up to HDSite.
1. Gessner et al., 2017: [publication](https://doi.org/10.1038/s41598-017-03922-3).
1. HDsite; [publication](https://doi.org/10.1073/pnas.1315532110).
1. PyHDX; [publication](https://doi.org/10.1021/acs.analchem.1c02155); [repository](https://github.com/Jhsmit/PyHDX); [web server](http://pyhdx.jhsmit.org/main).
1. Salmas et al., [publication](https://doi.org/10.1038/s42003-021-01709-x); [web server](https://hdxsite.nms.kcl.ac.uk/).
1. Saltzberg et al., [publication](https://doi.org/10.1021/acs.jpcb.6b09358) [part of IMP](https://github.com/salilab/imp)
1. Skinner et al., 2019: [publication](https://doi.org/10.1016/j.bpj.2019.02.024); [repository](https://github.com/skinnersp/exPfact).

## Analysis of HDX-MS data

HDX-MS data analysis methodology, focusing on HDX-MS data processing’s computational aspects:

1. Althaus et al., 2010: [Computing H/D-Exchange rates of single residues from data of proteolytic fragments](https://doi.org/10.1186/1471-2105-11-424).
1. Bai et al., 1993: [Primary structure effects on peptide group hydrogen exchange](https://doi.org/10.1002/prot.340170110).
1. Best et al., 2006: [Structural interpretation of hydrogen exchange protection factors in proteins: characterization of the native state fluctuations of CI2](https://doi.org/10.1016/j.str.2005.09.012).
1. Bradshaw et al., 2020: [Interpretation of HDX Data by Maximum-Entropy Reweighting of Simulated Structural Ensembles](https://doi.org/10.1016/j.bpj.2020.02.005).
1. Claesen et al., 2016: [Computational methods and challenges in hydrogen/deuterium exchange mass spectrometry](https://doi.org/10.1002/mas.21519).
1. Claesen et al., 2019: [POPPeT: a New Method to Predict the Protection Factor of Backbone Amide Hydrogens](https://doi.org/10.1007/s13361-018-2068-x).
1. Dovischenko et al., 2008: [Prediction of Residue Status to Be Protected or Not Protected From Hy-drogen Exchange Using Amino Acid Sequence Only](https://doi.org/10.2174/1874091x00802010077).
1. Englander et al., 1997: [Hydrogen exchange: the modern legacy of Linderstrøm-Lang.](https://doi.org/10.1002/pro.5560060517).
1. Fajer et al, 2012: [Improved Sequence Resolution by Global Analysis of Overlapped Peptides in Hydrogen/Deuterium Exchange Mass Spectrometry](https://doi.org/10.1007/s13361-012-0373-3).
1. Hageman et al, 2018: [Reliable Identification of Significant Differences in Differential Hydrogen Exchange-Mass Spectrometry Measurements Using a Hybrid Significance Testing Approach](https://doi.org/10.1021/acs.analchem.9b01325).
1. Harris et al, 2019: [Quantitative Evaluation of Native Protein Folds and Assemblies by Hydrogen Deuterium Exchange Mass Spectrometry (HDX-MS)](https://doi.org/10.1007/s13361-018-2070-3)   
1. Konermann et al., 2011: [Hydrogen exchange mass spectrometry for studying protein structure and dynamics](https://doi.org/10.1039/c0cs00113a).
1. Mazur et al., 2017: [The Area Between Exchange Curves as a Measure of Conformational Differences in Hydrogen-Deuterium Exchange Mass Spectrometry Studies](https://doi.org/10.1007/s13361-017-1615-1).  
1. Nguyen et al., 2018: [Reference Parameters for Protein Hydrogen Exchange Rates](https://doi.org/10.1007/s13361-018-2021-z).
1. Skinner et al., 2012: [Protein hydrogen exchange: testing current models](https://doi.org/10.1002/pro.2082).
1. Skinner et al., 2012: [Protein dynamics viewed by hydrogen exchange](https://doi.org/10.1002/pro.2081).
1. Tajoddin et al. 2020: [Analysis of Temperature-Dependent H/D Exchange Mass Spectrometry Experiments](https://doi.org/10.1021/acs.analchem.0c01828).
1. Zhang et al., 1993: [Determination of amide hydrogen exchange by mass spectrometry: a new tool for protein structure elucidation.](https://doi.org/10.1002/pro.5560020404).
1. Zhang et al., 2012: [Improved protein hydrogen/deuterium exchange mass spectrometry platform with fully automated data processing](https://doi.org/10.1021/ac300535r).
1. Zhang et al., 2020: [Complete Extraction of Protein Dynamics Information in Hydrogen/Deuterium Exchange Mass Spectrometry Data](https://doi.org/10.1021/acs.analchem.9b05724).

## Design of HDX-MS experiments

The methodology of HDX-MS and advances in HDX-MS experiments design:

1. Brown et al., 2017: [Bottom-up hydrogen deuterium exchange mass spectrometry: data analysis and interpretation](https://doi.org/10.1039/C7AN00662D).
1. Hamuro et al., 2020: [Tutorial: Chemistry of Hydrogen/Deuterium Exchange Mass Spectrometry](https://doi.org/10.1021/jasms.0c00260).
1. Masson et al., 2019: [Recommendations for performing, interpreting and reporting hydrogen deuterium exchange mass spectrometry (HDX-MS) experiments](https://doi.org/10.1038/s41592-019-0459-y).
1. Venable et al., 2012: [Subzero Temperature Chromatography for Reduced Back-Exchange and Improved Dynamic Range in Amide Hydrogen/Deuterium Exchange Mass Spectrometry](https://doi.org/10.1021/ac302488h).
1. Wei et al., 2014: [Hydrogen/deuterium exchange mass spectrometry for probing higher order structure of protein therapeutics: methodology and applications](https://doi.org/10.1016/j.drudis.2013.07.019).
1. Walters et al., 2012: [Minimizing back exchange in the hydrogen exchange-mass spectrometry experiment](https://dx.doi.org/10.1007%2Fs13361-012-0476-x).

## Applications of HDX-MS

Publications and pre-prints showcasing the applications of HDX-MS, where HDX-MS experiments play a crucial role:

1. Banos-Mateos et al., 2017: [High-fidelity DNA replication in Mycobacterium tuberculosis relies on a trinuclear zinc center](https://doi.org/10.1038/s41467-017-00886-w).
1. Filandrova et al., 2021: [Studying Protein–DNA Interactions by Hydrogen/Deuterium Exchange Mass Spectrometry](https://doi.org/10.1007/978-1-0716-1126-5_11).
1. Hamdi et al., 2017: [Structural disorder and induced folding within two cereal, ABA stress and ripening (ASR) proteins](https://doi.org/10.1038/s41598-017-15299-4).
1. Lim et al., 2017: [Epitope and Paratope Mapping Reveals Temperature-Dependent Alterations in the Dengue-Antibody Interface](https://doi.org/10.1016/j.str.2017.07.007).
1. Lesne et al., 2020: [Conformational maps of human 20S proteasomes reveal PA28- and immuno-dependent inter-ring crosstalks](https://doi.org/10.1038/s41467-020-19934-z). 
1. Masson et al., 2017: [Analysis of phosphoinositide 3-kinase inhibitors by bottom-up electron-transfer dissociation hydrogen/deuterium exchange mass spectrometry ](https://doi.org/10.1042/BCJ20170127).
1. Merkle et al., 2018: [Substrate-modulated unwinding of transmembrane helices in the NSS transporter LeuT](https://doi.org/10.1126/sciadv.aar6179).
1. Shukla et al., 2014: [Visualization of arrestin recruitment by a G-protein-coupled receptor](https://doi.org/10.1038/nature13430).
 
 
## General reviews

General reviews of HDX-MS experiments, not focusing on a single application, experimental problem, or computational method:

1. Engen et al., 2020: [Developments in Hydrogen/Deuterium Exchange Mass Spectrometry](https://doi.org/10.1021/acs.analchem.0c04281). 
1. Englander et al., 1997: [Hydrogen exchange: the modern legacy of Linderstrøm-Lang](https://doi.org/10.1002/pro.5560060517).
1. Englander et al., 1996: [Mechanisms and uses of hydrogen exchange](https://doi.org/10.1016/S0959-440X(96)80090-X).
1. Georgescauld et al., 2019: [Hydrogen deuterium exchange mass spectrometry applied to chaperones and chaperone-assisted protein folding](https://doi.org/10.1080/14789450.2019.1633920).
1. Giladi et al., 2020: [Hydrogen-Deuterium Exchange Mass-Spectrometry of Secondary Active Transporters: From Structural Dynamics to Molecular Mechanisms](https://doi.org/10.3389/fphar.2020.00070).
1. Hoofnagle et al., 2003: [Protein Analysis by Hydrogen Exchange Mass Spectrometry](https://doi.org/10.1146/annurev.biophys.32.110601.142417).
1. Konermann et al., 2011: [Hydrogen exchange mass spectrometry for studying protein structure and dynamics](https://doi.org/10.1039/C0CS00113A).
1. Mitra et al., 2020: [Emerging Role of Mass Spectrometry-Based Structural Proteomics in Elucidating Intrinsic Disorder in Proteins](https://doi.org/10.1002/pmic.202000011).
1. Narang et al., 2020: [HDX-MS: An Analytical Tool to Capture Protein Motion in Action](https://doi.org/10.3390/biomedicines8070224).

## HDX-MS datasets

- SAP, Enolase, Barnase, Alpha_lac [Salmas et al](https://doi.org/10.1038/s42003-021-01709-x), [link](https://hdxsite.nms.kcl.ac.uk/static/Example_Modeller_.zip)
- SecB [Smit et al](https://doi.org/10.1101/2020.09.30.320887), [link](https://raw.githubusercontent.com/Jhsmit/PyHDX/master/tests/test_data/ecSecB_apo.csv)
- SNase [Gessner et al](https://doi.org/10.1038/s41598-017-03922-3), [link](https://static-content.springer.com/esm/art%3A10.1038%2Fs41598-017-03922-3/MediaObjects/41598_2017_3922_MOESM2_ESM.xls)
- Human 20S proteasomes [Lesne et al](https://doi.org/10.1038/s41467-020-19934-z), [link](https://www.omicsdi.org/dataset/pride/PXD018921)
- Fab fragment of NISTmAb [Hudgens et al (1)](https://doi.org/10.6028/jres.124.009) [Hudgens et al (2)](https://pubs.acs.org/doi/10.1021/acs.analchem.9b01100), [link](https://data.nist.gov/od/id/76F046F18CCA46FBE05324570681CB301977)
- XylE, LacY, GlpT [Martens et al](https://dx.doi.org/10.1038%2Fs41467-018-06704-1), [link](https://www.ebi.ac.uk/pride/archive/projects/PXD011060)
- HalM2 [Habibi et al](https://doi.org/10.1021/jacs.9b06020), [link](https://figshare.com/articles/dataset/Insights_into_the_Dynamic_Structural_Properties_of_a_Lanthipeptide_Synthetase_using_Hydrogen_Deuterium_Exchange_Mass_Spectrometry/9778637/1)
- Human Hb, Human IgG4 [Tsiatsiani et al](https://doi.org/10.1021/acs.analchem.7b01161), [link](https://pubs.acs.org/doi/suppl/10.1021/acs.analchem.7b01161/suppl_file/ac7b01161_si_003.xlsx)
- SecYEG, SecA [Ahdash et al](https://doi.org/10.7554/eLife.47402), [link](https://www.omicsdi.org/dataset/pride/PXD013594)
- VDR, RORγ [Saltzberg et al](https://doi.org/10.1021/acs.jpcb.6b09358), [link](https://doi.org/10.1021/acs.jpcb.6b09358.s001)
- 15-LOX [Droege et al](https://doi.org/10.1021/acs.biochem.7b00559) [link](https://doi.org/10.1021/acs.biochem.7b00559.s002)
- human Rag GTPase [Anandapadamanaban et al](http://doi.org/10.1126/science.aax3939) [link](https://www.ebi.ac.uk/pride/archive/projects/PXD013051)
- Eukaryotic Ribosome GCN2 [Inglis AJ et al](http://doi.org/10.1073/pnas.1813352116) [link](https://www.ebi.ac.uk/pride/archive/projects/PXD015778)

